requirements.txt

```diff
@@ -8,7 +8,6 @@ static3==0.5.1
 wsgiref==0.1.2
 -e git+https://github.com/funkybob/django-flatblocks.git#egg=django-flatblocks
 sendgrid-django
-psycopg2==2.5.4
```

Comment:
@brittainhard: 
I wasn't able to locate the traceback since it was a while ago. Currently my conda list shows that `psycopg2` is on 2.7.1. Would this be a problem for heroku deployment?